src/style.css

```diff
@@ -0,0 +1,134 @@
+body {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.stars--0 {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: flex-end;
+}
+
+.stars__star {
+  display: flex;
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  margin: 0 2px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: flex-end;
+}
+
+.stars--2 {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: flex-end;
+}
+
+.stars--3 {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: flex-end;
+}
+
+.stars--4 {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: flex-end;
+}
+
+.stars--5 {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: flex-end;
+}
+
+.stars--0 > .stars__star:nth-of-type(1n + 6) {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star-active.svg);
+  margin: 0 2px;
+  background-repeat: no-repeat;
+  display: flex;
+
+}
+
+.stars--1 > .stars__star:nth-of-type(1n + 5) {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star-active.svg);
+  margin: 0 2px;
+  background-repeat: no-repeat;
+  display: flex;
+
+}
+
+.stars--2 > .stars__star:nth-of-type(1n + 4) {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star-active.svg);
+  margin: 0 2px;
+  background-repeat: no-repeat;
+  display: flex;
+
+}
+
+.stars--3 > .stars__star:nth-of-type(1n + 3) {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star-active.svg);
```

Comment:
use common styles to one selector, and only different style to exact selector, and combine selectors
```
.stars--1 > .stars__star:nth-last-child(n+5),
.stars--2 > .stars__star:nth-last-child(n+4),
.stars--3 > .stars__star:nth-last-child(n+3),
.stars--4 > .stars__star:nth-last-child(n+2),
.stars--5 > .stars__star:nth-last-child(n+1) {
  background-image: url(./images/star-active.svg);
}
```